tests/datasets/test__init__.py

```diff
@@ -0,0 +1,424 @@
+# Copyright 2018-2019 Faculty Science Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pytest
+
+from faculty import datasets
+
+
+@pytest.fixture
+def mock_client(mocker):
+    session = mocker.Mock()
+    get_session_mock = mocker.patch(
+        "faculty.datasets.get_session", return_value=session
+    )
+
+    project_id = mocker.Mock()
```

Comment:
It seems a bit odd that this is created by the `mock_client` fixture, which otherwise does not seem to depend on project ID.

We could alternatively just have a single `uuid.UUID` in the module namespace used by all tests:

```
import uuid

PROJECT_ID = uuid.uuid4()
```